clients/cloud/scala/src/main/scala/io/confluent/examples/clients/scala/Producer.scala

```diff
@@ -0,0 +1,26 @@
+package io.confluent.examples.clients.scala
+
+import java.util.Properties
+import com.typesafe.config.{Config, ConfigFactory}
+import org.apache.kafka.clients.producer.{KafkaProducer, ProducerRecord}
+
+object Producer extends App {
+  var conf = ConfigFactory.load("ccloud.properties")
+  val props = propsFromConfig(conf)
+  val producer = new KafkaProducer[String, String](props)
+  for( i <- 1 to 10) {
+    val record = new ProducerRecord[String, String]("testtopic", "key1", "val"+i)
+    producer.send(record)
+  }
+  producer.close()
```

Comment:
Should do a flush() and preferably also check the send statuses (through callback or future)

Reply:
done